src/main/java/hu/oe/nik/szfmv/automatedcar/virtualfunctionbus/packets/HMIOutputPackets/BaseInputPacket.java

```diff
@@ -0,0 +1,17 @@
+package hu.oe.nik.szfmv.automatedcar.virtualfunctionbus.packets.HMIOutputPackets;
+
+public interface BaseInputPacket {
+
+    int getGasPedalValue();
+    int getBreakPedalValue();
+    int getSteeringWheelValue();
+    int getIndexValue(/*negatív -> balra, pozitív -> jobbra, 0 -> kikapcsolva */);
+    char getShiftValue();
```

Comment:
Mindenképp `char`?

Reply:
Ha a `getIndexValue` csak 3 értéket vehet fel, akkor szerintem használjatok inkább enum-ot, pl.:
```java
public enum IndexState {
    LEFT,
    RIGHT,
    NONE
}

IndexState getIndexState();
```

Reply:
Elvileg lehet egy negyedik is: BOTH, bár nem hiszem, hogy különösebb szükség lenne erre ebben a szoftverben.